docs/undergrad.yaml

```diff
@@ -365,13 +364,13 @@ Single Variable Real Analysis:
 # 7.
 Single Variable Complex Analysis:
   Complex Valued series:
-    radius of convergence:
-    properties of sums of complex valued series on their disks of convergence:
-    continuity:
+    radius of convergence: 'formal_multilinear_series.radius'
+#    properties of sums of complex valued series on their disks of convergence: (this is a heading?)
+    continuity: 'has_fpower_series_on_ball.continuous_on'
     differentiability with respect to the complex variable:
     antiderivative:
-    complex exponentials:
-    extension of circular functions to the complex plane:
+    complex exponential:
```

Comment:
Why don't we write `complex.exp` here?

Reply:
I noticed the omission but assumed you'd also noticed it and omitted it for a reason :)

(The reason that imaginary-Patrick-in-my-head gave for the omission was that although `complex.exp` and `complex.sin` exist and are constructed by power series, this is done in a one-off ad hoc way rather than by using the library on analytic functions.)

Reply:
Honestly, I don't remember. Maybe I felt it was silly to have a complex analysis section containing only this. In any case I think we can add this, so I'll do it.